src/components/content/exercises/exercise.tsx

```diff
@@ -44,10 +49,33 @@ export function Exercise({ node }: ExerciseProps) {
 
       {renderToolsAndLicense()}
 
-      {solutionVisible && renderSolutionBox()}
+      {solutionVisible && renderSolution()}
     </Wrapper>
   )
 
+  function renderSolution() {
+    return (
+      <Solution
+        solutionEdtrState={node.solutionEdtrState}
+        solutionLegacy={node.solutionLegacy}
+        license={node.solutionLicense}
+        authorTools={
```

Comment:
Ok, we would need to copy auth and loaded to the new solution component ... this way, single view solutions will also have an edit box.

Reply:
I did not want to support that since the edit workflow we discussed (loading whole exes or even groups). 
I'd actually discourage the entity view of a solution. This is manly to support the injection edge case.

Reply:
How do we do that with events atm? Can I single view an event?

Reply:
Ah, we could redirect the single view of an id to the single view of the parent exercise –
that should work with the context id's right?